keras_cv/layers/preprocessing/random_resized_crop_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+from keras_cv.layers import preprocessing
+
+
+class RandomResizedCropTest(tf.test.TestCase):
+    def test_train_augments_image(self):
+        # Checks if original and augmented images are different
+
+        input_image_shape = (4, 300, 300, 3)
+        image = tf.random.uniform(shape=input_image_shape)
+
+        layer = preprocessing.RandomResizedCrop(
+            target_size=(224, 224), area_factor=(0.8, 1.0)
+        )
+        output = layer(image, training=True)
+
+        output_image_resized = tf.image.resize(output, input_image_shape)
+
+        self.assertNotEqual(image, output_image_resized)
+
+    def test_preserves_image(self):
```

Comment:
add a grayscale example as @sayakpaul suggested, then I think I feel pretty good about this PR!  

Perhaps a seeded random behavior test would be nice too - something like sample random noise as an image batch: (2, 512, 512, 3), crop randomly (200, 200), then assert that they are not equal.  With proper seeding this should work fine.

Reply:
I guess `test_train_augments_image` already does that.